entity-types/infra-mysqlnode/golden_metrics.yml

```diff
@@ -7,6 +7,10 @@ queriesPerSecond:
       from: MysqlSample
       eventId: entityGuid
       eventName: entityName
+    # defaulting to newRelic query for now since we cannot match value functions for OpenTelemetry cumulative metrics.
```

Comment:
I would prefer to not have commented code, maybe just leave a comment like no custom query for openTelemetry becasue we cannot match...

Reply:
Also could you develop a little more why the commented code can't be uncommented.
Is the query.queriesPerSecond available when ingesting from OTEL?

Reply:
@lujop `query.queriesPerSecond` is not available but instead `mysql.query.slow.count` is available. 
However, they are not directly compatible with current golden metrics. 
Please refer to [this](https://github.com/newrelic/entity-definitions/pull/1347#discussion_r1453490456) comment earlier.